README.md

```diff
@@ -328,6 +398,34 @@ $ gem install blueprinter
 
 You should also have `require 'json'` already in your project if you are not using Rails or if you are not using Oj.
 
+## Sorting
+By default the response sorts the keys by name. If you want the fields to be sorted in the order of definition, use the below configuration option.
+
```

Comment:
If you could just document the options available here, I think that's all I'd request and looks good to me. Something like:
`sort_fields_by ` options:
  - `:name_asc` (default): Sorts the fields in ascending order by field name.
  - `:definition`: Preserves the order the fields were defined in.

Reply:
:name_asc is default. That's why I skipped it and gave an example for :definition alone.